barista-kafka/package.json

```diff
@@ -4,12 +4,21 @@
   "description": "A clone of https://github.com/cescoffier/reactive-coffeeshop-demo in JS",
   "main": "index.js",
   "scripts": {
-    "start": "node .",
+    "start": "NODE_OPTIONS='--require ./tracing.js' node .",
     "lint": "eslint ."
   },
   "author": "Red Hat, Inc.",
   "license": "Apache-2.0",
   "dependencies": {
+    "@opentelemetry/api": "^1.2.0",
+    "@opentelemetry/instrumentation": "^0.32.0",
+    "@opentelemetry/instrumentation-fastify": "^0.30.0",
+    "@opentelemetry/instrumentation-http": "^0.33.0",
+    "@opentelemetry/resources": "^1.6.0",
+    "@opentelemetry/sdk-trace-base": "^1.6.0",
+    "@opentelemetry/sdk-trace-node": "^1.6.0",
+    "@opentelemetry/semantic-conventions": "^1.6.0",
+    "opentelemetry-instrumentation-kafkajs": "^0.32.0",
     "chance": "^1.1.7",
     "dotenv": "^10.0.0",
     "kafkajs": "^1.15.0",
```

Comment:
Please keep in mind to update kafkajs to 1.16.0 :+1: 
See the comment in the jira issue